### PR TITLE
Feature: rake tasks for drop accounting and extract configs

### PR DESCRIPTION
**Pull request by ymasiuk**

### CODE REVIEW

File: lib/tasks/database.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+
+namespace :database do
```
Comment (mod) [inline lib/tasks/database.rake]: maybe

clear:accounting

File: lib/tasks/clear.rake [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+
+namespace :clear do
+  desc 'Clear accounting.'
```
Comment (ysv) [inline lib/tasks/clear.rake]: could you write more explicit description because this one is not clear for me

File: lib/tasks/clear.rake [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  task accounting: :environment do
+    table_names = ['accounts', 'adjustments', 'assets', 'beneficiaries',
```
Comment (ysv) [inline lib/tasks/clear.rake]: accounting_tables = %w[accounts adjustments ...]

File: lib/tasks/export.rake [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    Rake::Task["export:blockchains"].invoke
+    Rake::Task["export:currencies"].invoke
+    Rake::Task["export:markets"].invoke
+    Rake::Task["export:wallets"].invoke
+    Rake::Task["export:trading_fees"].invoke
+  end
+
+  desc 'Export blockchains config to yaml file.'
+  task blockchains: :environment do
+    result = ActiveRecord::Base.connection.exec_query("SELECT * FROM blockchains").to_hash
```
Comment (ysv) [inline lib/tasks/export.rake]: `Blockchain.all.map { |b| b.attributes.except('created_at', 'updated_at') }` - use this variant for other models also

File: lib/tasks/export.rake [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+    result.each do |b|
+      b.except!('created_at', 'updated_at')
+    end
+    File.open("config/seed/blockchains.yml","w") do |file|
+       file.write result.to_yaml
```
Comment (ysv) [inline lib/tasks/export.rake]: Make sure that generated yml is human readable

File: lib/tasks/export.rake [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  end
+
+  desc 'Export blockchains config to yaml file.'
+  task blockchains: :environment do
+    result = ActiveRecord::Base.connection.exec_query("SELECT * FROM blockchains").to_hash
+
+    result.each do |b|
+      b.except!('created_at', 'updated_at')
+    end
+    File.open("config/seed/blockchains.yml","w") do |file|
```
Comment (ysv) [inline lib/tasks/export.rake]: I'm not sure that we want to rewrite existing blockchains.yml

File: lib/tasks/export.rake [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  end
+
+  desc 'Export wallets config to yaml file.'
+  task wallets: :environment do
+    result = ActiveRecord::Base.connection.exec_query("SELECT * FROM wallets").to_hash
+
+    result.each do |w|
+      w.except!('created_at', 'updated_at')
+      w.transform_values! { |v| v.is_a?(BigDecimal) ? v.to_f : v }
+      w['settings'] = Wallet.find_by(address: w['address']).settings
```
Comment (ysv) [inline lib/tasks/export.rake]: looks strange for me

File: lib/tasks/export.rake [hunk 8]
```diff
@@ -0,0 +1,91 @@
+# encoding: UTF-8
```
Comment (ysv) [inline lib/tasks/export.rake]: as far as I can see the process of exporting is pretty same for all models so I advice to make single method which receive argument(s)
```ruby
def export(model_name)
  # 1. select all objects and store them as array of hashes
  # 2. remove timestamp columns
  # 3. format BigDecimal values
end
```

**Review by ysv — CHANGES_REQUESTED** [review]

File: lib/tasks/clear.rake (hunk 2 above)
Comment (dnfd) [reply lib/tasks/clear.rake]: Nice pun :D

**Review by mnaichuk — APPROVED** [review]

**Review by ysv — APPROVED** [review]

**Review by mod — APPROVED** [review]

**Pull request merged**